packages/frontend/core/src/components/attachment-viewer/attachment-embed-preview.tsx

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from 'react';
+
+import { AudioBlockEmbedded } from './audio/audio-block';
+import { AttachmentPreviewErrorBoundary } from './error';
+import { PDFViewerEmbedded } from './pdf/pdf-viewer-embedded';
+import type { AttachmentViewerProps } from './types';
+import { getAttachmentType } from './utils';
+
+// In Embed view
+export const AttachmentEmbedPreview = ({ model }: AttachmentViewerProps) => {
```

Comment:
I personally prefer to put components in blocksuite under the `src/blocksutie` folder.